utils/email.py

```diff
@@ -0,0 +1,54 @@
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils import timezone
+from smtpapi import SMTPAPIHeader
+import datetime
+
+
+def send_template_email(
+        to_email,
+        subject,
+        template,
+        context,
+        sent_by=None,
+        hours=None):
```

Comment:
we like to do indentation on function *definitions* different from function *calls* -- this is correct for a function call that's longer than 79 characters wide but we want the following for function definitions:
```
def send_template_email( 
        to_email, subject, template, context, sent_by=None,
        hours=None):
    body = render_to_string(template, context)
    ...
```